src/main/resources/application.conf

```diff
@@ -12,7 +18,17 @@ akka.management {
 
       # pick the discovery method you'd like to use:
       discovery-method = kubernetes-api
+
+      required-contact-point-nr = ${REQUIRED_CONTACT_POINT_NR}
```

Comment:
this was in the guide https://developer.lightbend.com/guides/openshift-deployment/lagom/forming-a-cluster.html#cluster-bootstrap